backend/content-module/services/serviceContent.ts

```
@@ -469,112 +489,125 @@ static async shareContent(contentID: string, userId: string) {
       const userData = userDoc.data();
       const likedContent = userData.likedContent || [];
       const following = userData.following || [];
 
       const allContent = await ContentService.getAllContent();
-      
+
       // Extract scoring logic to separate methods for better maintainability
-      const scoredContent = allContent.map(content => {
+      const scoredContent = allContent.map((content) => {
         // Calculate different types of scores
         const creatorScore = this.calculateCreatorScore(content, following);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)